Update docs and add editorial board page (#103)
python/docs-community: docs/community/community-guide.md

There are a few resources that are particularly useful for team members. Here's
a quick list to get you started.

- [devguide](https://devguide.python.org)
- [docs](https://docs.python.org)
- python_docs_theme

> - discourse channel

Reviewer: ```suggestion
- [discourse category](https://discuss.python.org/c/documentation/26)
```

> - python_docs_theme
> - discourse channel

Reviewer 2: ```suggestion
- [Python Docs Sphinx Theme](https://github.com/python/python-docs-theme)
- [Discourse category](https://discuss.python.org/c/documentation/26)
```

> - discord channel

Reviewer 2: ```suggestion
- Discord server
```

### General information about communication channels

We are trying to organize our discussions in order to help both contributors and
maintainers find and choose the right communication channels and have a positive experience :-)

In this respect, we are using:

1. GitHub issues for specific discussions related to changing a repository's content.

> 2. The [Discourse forum](http://discuss.python.org/) for general discussions, support

Reviewer: ```suggestion
2. The [Discourse forum](https://discuss.python.org/c/documentation/26) for general discussions, support
```
